buttercup.el

```diff
@@ -560,7 +579,12 @@ KEYWORD can have one of the following values:
                                  :return-value return-value
                                  :current-buffer (current-buffer)))
               return-value)))
-    this-spy-function))
+    ;; Add the interactive form from `fake-function', if any
+    (if (interactive-form fake-function)
+        `(lambda (&rest args)
+           ,(interactive-form fake-function)
+           (apply ',this-spy-function args))
```

Comment:
No, because `this-spy-function` is a variable containing a function (i.e. we need `(symbol-value 'this-spy-function)`. And `args` doesn't need un-quoting.